lcc_comchain_base/services/comchain_services.py

```diff
@@ -74,3 +74,24 @@ def activate(self, params):
             partners.activateComchainUser(account)
 
         return True
+
+    @restapi.method(
+        [(["/credit"], "POST")],
+        input_param=Datamodel("comchain.credit.info"),
+        output_param=Datamodel("comchain.credit.response"),
+    )
+    def credit(self, params):
+        """
+        Credit user account with amount, and generate accounting entry
+        """
```

Comment:
minor: not PEP257 https://peps.python.org/pep-0257/ it is best on one line ( https://peps.python.org/pep-0257/#one-line-docstrings )